service_mesh/servicemesh-release-notes.adoc

```diff
@@ -5,8 +5,6 @@ include::modules/ossm-document-attributes.adoc[]
 
 toc::[]
 
-= {RN_BookName}
```

Comment:
@JStickler Can you review this? I removed this since it was adding a second level 1 heading in this file which was causing a build error.

Reply:
LGTM.  I thought we'd already removed that, sorry.  =(